test/form/persist-files.test.js

```diff
@@ -183,3 +213,9 @@ describe('Submission journey test', () => {
 /**
  * @import { Server } from '@hapi/hapi'
  */
+
+/**
+ * @typedef {import('@defra/forms-model').FormMetadataAuthor} FormMetadataAuthor
+ * @typedef {import('@defra/forms-model').FormMetadataState} FormMetadataState
+ * @typedef {import('@defra/forms-model').FormMetadata} FormMetadata
+ */
```

Comment:
Mind if we stick to the new `@import` syntax?

```suggestion
/**
 * @import { FormMetadata, FormMetadataAuthor, FormMetadataState } from '@defra/forms-model'
 * @import { Server } from '@hapi/hapi'
 */
```

Reply:
Resolved with ca3daa7fa942f234b3f6e88d4e981b0bdec7a14a